study.md

```diff
@@ -48,7 +49,7 @@ in Rails.
 In your own words, define what the router does in Rails.
 
 ```md
-<!-- your answer here -->
+<!-- The router is the component that helps generate urls for the corrisponding ajax request to the backend. This part also interacts with the controller layer. --guides.rubyonrails.org/routing.html-->
```

Comment:
Yes, though it "generates" urls by defining them, what really happens is that it generates definitions of acceptable URLs and request methods.